test/tsconfig.json

```diff
@@ -8,7 +8,8 @@
 		},
 		"strict": false,
 		"allowJs": true,
-		"types": ["mocha", "node"]
+		"types": ["mocha", "node"],
+		"target": "es2020"
```

Comment:
To prevent future surprises we should probably check that all of the features this enables are supported in the node versions we run these tests against.